coalib/bearlib/languages/definitions/ApacheConf.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class ApacheConf:
+    extensions = '.htaccess', 'apache.conf', 'apache2.conf'
```

Comment:
IMO yes, my general opinion on those kinds of issues is they have probably thought longer on it and have more experience. This seems sensible to me.